app/screens/channel/index.tsx

```diff
@@ -3,15 +3,39 @@
 
 import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
 import withObservables from '@nozbe/with-observables';
+import {combineLatest, of as of$} from 'rxjs';
+import {switchMap} from 'rxjs/operators';
 
+import {observeCallsConfig, observeChannelsWithCalls, observeCurrentCall} from '@calls/state';
+import {withServerUrl} from '@context/server';
 import {observeCurrentChannelId} from '@queries/servers/system';
 
 import Channel from './channel';
 
 import type {WithDatabaseArgs} from '@typings/database/database';
 
-const enhanced = withObservables([], ({database}: WithDatabaseArgs) => ({
-    channelId: observeCurrentChannelId(database),
-}));
+type EnhanceProps = WithDatabaseArgs & {
+    serverUrl: string;
+}
 
-export default withDatabase(enhanced(Channel));
+const enhanced = withObservables([], ({database, serverUrl}: EnhanceProps) => {
+    const channelId = observeCurrentChannelId(database);
+    const isCallsPluginEnabled = observeCallsConfig(serverUrl).pipe(
+        switchMap((config) => of$(config.pluginEnabled)),
+    );
+    const isCallInCurrentChannel = combineLatest([channelId, observeChannelsWithCalls(serverUrl)]).pipe(
+        switchMap(([id, calls]) => of$(Boolean(calls[id]))),
+    );
+    const isInCall = observeCurrentCall().pipe(
+        switchMap((call) => of$(Boolean(call))),
+    );
+
+    return {
+        channelId,
+        isCallsPluginEnabled,
```

Comment:
you don't really need to pass this as a prop, you could instead have `isInCall` and `isCallInCurrentChannel` use this and return `false` if not enabled